src/AzureClient/AzureClient.cs

```diff
@@ -176,57 +187,89 @@ public async Task<ExecutionResult> GetConnectionStatusAsync(IChannel channel)
                 return AzureClientError.NoOperationName.ToExecutionResult();
             }
 
-            var machine = Azure.Quantum.QuantumMachineFactory.CreateMachine(ActiveWorkspace, ActiveTarget.TargetName, ConnectionString);
+            var machine = QuantumMachineFactory.CreateMachine(ActiveWorkspace, ActiveTarget.TargetName, ConnectionString);
             if (machine == null)
             {
                 // We should never get here, since ActiveTarget should have already been validated at the time it was set.
                 channel.Stderr($"Unexpected error while preparing job for execution on target {ActiveTarget.TargetName}.");
                 return AzureClientError.InvalidTarget.ToExecutionResult();
             }
 
-            var operationInfo = operationResolver.Resolve(operationName);
-            var entryPointInfo = new EntryPointInfo<QVoid, Result>(operationInfo.RoslynType);
-            var entryPointInput = QVoid.Instance;
+            channel.Stdout($"Submitting {operationName} to target {ActiveTarget.TargetName}...");
 
-            if (execute)
+            IEntryPoint? entryPoint = null;
```

Comment:
Nice that we are embracing nullable reference types. :)

Reply:
Yes! @cgranade has been evangelizing this. We are trying to include `#nullable enable` at the top of all new C# files, and gradually add them to the old files.